distribution/kernel/carbon-home/repository/resources/conf/templates/repository/conf/carbon.xml.j2

```diff
@@ -573,7 +580,7 @@
         <!--
            The total file upload size limit in MB
         -->
-        <TotalFileSizeLimit>100</TotalFileSizeLimit>
+        <TotalFileSizeLimit>server.file_upload_config.total_file_size_limit</TotalFileSizeLimit>
```

Comment:
Add the key inside {{server.file_upload_config.total_file_size_limit}}

Reply:
fixed